src/vs/workbench/services/textmodelResolver/common/textModelResolverService.ts

```diff
@@ -47,7 +47,7 @@ class ResourceModelCollection extends ReferenceCollection<TPromise<ITextEditorMo
 			} else {
 				model.dispose();
 			}
-		});
+		}, () => void 0);
```

Comment:
Not sure if we should log here or add some other handler. We do add catch callbacks  at higher levels that do error handling when the file does not exist. These still should be invoked after this change